policyengine_us/variables/gov/states/la/tax/income/credits/cdcc/la_cdcc_non_refundable.py

```diff
@@ -0,0 +1,29 @@
+from policyengine_us.model_api import *
+
+
+class la_cdcc_non_refundable(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Louisiana non-refundable Child and Dependent Care Credit"
+    unit = USD
+    definition_period = YEAR
+    reference = "http://legis.la.gov/Legis/Law.aspx?d=101769"
+    defined_for = StateCode.LA
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.states.la.tax.income.credits.cdcc
+        us_agi = tax_unit("adjusted_gross_income", period)
+        # determine LA nonrefundable cdcc amount
+        la_cdcc_pre_cap = tax_unit("la_cdcc", period)
+
+        upper_bracket = (
```

Comment:
move all this logic into `la_cdcc`

this variable doesn't need a formula, just
```
adds = ["la_cdcc"]
subtracts = ["la_cdcc_refundable"]
```